### DISCUSSION TITLE
Now Deploy with Go help needed

### DISCUSSION THREAD

**Discussion by rabet** [vercel/vercel#4050] [Help] [answered]
Hey guys,
I need to deploy the awesome Gotalk (https://github.com/rsms/gotalk) websocket chat on Now Go runtime. But I am not sure how to handle the server websocket initialization - https://github.com/rsms/gotalk/blob/master/examples/websocket-chat/server.go (end of a file - ```gotalkws := gotalk.WebSocketHandler()
   gotalkws.OnAccept = onAccept
   http.Handle("/gotalk/", gotalkws)```
 ) with Now's HandlerFunc. I have index.html in root which automatically displays but i need to setup the websocket conn. on /gotalk route somehow.
Many thanks for any help!

**Comment by coetry** [comment, marked as answer]
We don't support websockets on our platform since they require long-lived TCP connections. Our Serverless Functions have a timeout constraint and are designed to return HTTP requests as fast as possible.

**Reply by styfle** [reply]
This limitation is documented here: https://vercel.com/docs/v2/platform/limits#websockets

**Reply by rabet** [reply]
Understood, thanks for the answer.
